include/ignition/math/Plane.hh

```diff
@@ -157,8 +157,8 @@ namespace ignition
         Vector3<T> rotatedXAxis = rotation * Vector3<T>::UnitX;
         Vector3<T> rotatedYAxis = rotation * Vector3<T>::UnitY;
 
-        auto xBasis = rotatedXAxis.VectorProjectionLength(intersection);
-        auto yBasis = rotatedYAxis.VectorProjectionLength(intersection);
+        auto xBasis = rotatedXAxis.Dot(intersection);
+        auto yBasis = rotatedYAxis.Dot(intersection);
```

Comment:
This was dividing by length before

Reply:
Yes. But the length of the unit vectors is 1.

Reply:
Ah yes, I hadn't noticed that